WORKSPACE

```diff
@@ -57,13 +57,12 @@ android_workspace()
 #    name = "androidndk",
 #)
 
-# use Neuron Delegate aar before we have updated source code
-#http_archive(
-#    name = "neuron_delegate",
-#    sha256 = "2e4600c99c9b4ea7a129108cd688419eeef9b2aeabf05df6f385258e19ca96c4",
-#    strip_prefix = "tflite-neuron-delegate-2.6.0",
-#    urls = ["https://github.com/MediaTek-NeuroPilot/tflite-neuron-delegate/archive/v2.6.0.tar.gz"],
-#)
+http_archive(
+    name = "neuron_delegate",
+    sha256 = "2bef00cc7ba6f649a437e979fe1b36bdd94f575b5c00e0d08cb4e92e586f49a7",
+    strip_prefix = "tflite-neuron-delegate-2.8.0",
+    urls = ["https://github.com/MediaTek-NeuroPilot/tflite-neuron-delegate/archive/refs/tags/v2.8.0.tar.gz"],
+)
```

Comment:
We should also delete files from `mobile_back_tflite/cpp/backend_tflite/neuron` that we no longer need: `README.md` and `tensorflow-lite-neuron.aar`

Reply:
done